### ISSUE TITLE
Overall Occupancy falsely clear, Target Active, Zone x Detected and zone x target count > 0

### ISSUE THREAD

**Issue by clintebbesen** [EverythingSmartHome/everything-presence-lite#240]
This time the overall occupancy is clear when it should be detected, but everything else is working as expected. Zone 1 how's as detected, it shows 1 target, and target 1 active is on

![image](https://github.com/user-attachments/assets/9569fc49-9456-41e2-a91f-0977874ff2bc)
![image](https://github.com/user-attachments/assets/9b10ae77-2c00-457d-950a-73adb5ba3de0)

**Comment by jwvdhelm** [comment]
I do have this same issue, Target 1 is active and measures Distance and shows me on the X and Y axel, but the occupancy in all my 3 zones are clear and target count is 0
all was well yesterday 14-11-2024 at the day time but after 22:00 is broke. dont realy know what updates i did.
to troubleshoot i reinstalled the ESPconfig, with no result. Reboot the EPL, no result. 
not sure what i can do more.
![image](https://github.com/user-attachments/assets/8583ba1a-f850-464e-8bf8-d8edc41a7916)
![Screenshot 2024-11-15 000818](https://github.com/user-attachments/assets/f275c7d2-989c-443d-aa88-f1e30aa1cf0f)

**Comment by EverythingSmartHome** [comment]
Can you show the config for all zones and exclusion zone?

**Comment by jwvdhelm** [comment]
Hi Lewis, thank you for reacting. is this what you want to see?
![Screenshot 2024-11-15 083154](https://github.com/user-attachments/assets/c275564c-8d7f-40a7-8dfb-99df58803eac)
![Screenshot 2024-11-15 083209](https://github.com/user-attachments/assets/345bc631-e220-415f-bce4-28857f16b43b)

**Comment by EverythingSmartHome** [comment]
Yes thanks, in this case you have an exclusion zone configured (Occupancy mask coordinates in your screenshot above) meaning that this is working as intended. Your targets coordinates is inside the exclusion zone so while it's still tracking the target, it will show occupancy as not detected (as that's the intended behaviour of the exclusion zone).

If you don't want this to happen, set the occupancy mask to 0 again which is the default

**Comment by jwvdhelm** [comment]
how wauw that was stupid, thanks for this hint.....it is oke now. thank you so much.
Keep up the good work it is really great and is much appreciated.
I like your youtube and written guidance
